package.json

```diff
@@ -3,9 +3,13 @@
   "version": "0.0.0-dev",
   "description": "JS implementation of dag-cbor",
   "type": "module",
+  "types": "dist/index.d.ts",
```

Comment:
@rvagg shouldn't this be `index.d.ts` instead since `dist` gets published instead ?